libs/platform/table/components/table-p13-dialog/columns/columns.component.html

```diff
@@ -75,8 +75,8 @@ <h4 fd-title [headerSize]="4">
                 (click)="_moveActiveToBottom()"
             ></button>
         </fd-toolbar>
-        <ul fd-list>
-            <li fd-list-item>
+        <ul fd-list [selection]="true" aria-multiselectable="true">
```

Comment:
`aria-multiselectable` is used in roles grid, listbox, tablist and tree. Your `ul` element should have one of these roles. Additionally, the list elements should be with role `option` and `aria-selected` set to true or false.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable

Reply:
https://www.digitala11y.com/aria-multiselectable-properties/

Reply:
@InnaAtanasova the correct roles and aria-selected are being applied to the elements via the directives. If you inspect the html:
![Screenshot 2024-01-02 at 5 51 54 PM](https://github.com/SAP/fundamental-ngx/assets/2471874/f3ed3a11-360c-4d34-9055-27a51c5b041b)

Reply:
maybe we should make `multiselectable` a prop of the directive in this case. Could be in a separate PR. I will approve this one.